docs/core/plugins/sieve.md

```diff
@@ -46,128 +58,191 @@ protocol lmtp {
+of the scripts in that directory is determined by the file names, using a normal
+8-bit per-octet comparison.
 
-The `file` location is used to retrieve Sieve scripts from the file system.
-This is the default type if the type specifier is omitted from the location
-specification.
-
-The location can either point to a directory or to a regular file. If the
-location points to a directory, a script called `name` is retrieved by
-reading a file from that directory with the file name `name.sieve`.
-
-When this location type is involved in a [[setting,sieve_before]] or
-[[setting,sieve_after]] script sequence, and the location points to a
-directory, all files in that directory with a `.sieve` extension are part of
-the sequence. The sequence order of the scripts in that directory is
-determined by the file names, using a normal 8bit per-character
-comparison.
-
-Unless overridden using the `;bindir=<path>` location option, compiled
-binaries for scripts retrieved from the `file` location type are by
+Unless overridden using the [[setting,sieve_script_bin_path]] setting, compiled
+binaries for scripts retrieved from a `file` script storage are by
 default stored in the same directory as where the script file was found
 if possible.
 
 #### Configuration
 
-The script location syntax is specified as follows:
-
-```
-location = file:<path>[;<option>[=<value>][;...]]
-```
+The `file` storage driver supports all settings described in
+[Common Settings](#common-settings). Additionally, the following settings apply
+to this driver:
 
-The location `<path>` is a file system path pointing to a directory
-containing one or more script files with names structured as
-`<script-name>.sieve` with the active option (default `~/.dovecot.sieve`)
-specifying a symlink to the one that will be used, or without the active
-option specified, it may be a script file instead of a directory.
+<SettingsComponent tag="sieve-storage-file" />
 
-##### Settings
-
-This location supports all settings described in
-[Common Settings](#common-settings).
-
-##### `bindir`
-
-If the `bindir` setting is omitted, the binary is stored in the same
-directory as where the script file was found, if possible
-
-###### `active=<path>`
-
-When [[link,managesieve_server]] is used, one script in the storage can
-be active; i.e., evaluated at delivery.
-
-For the `file` location type, the active script in the storage directory
-is pointed to by a symbolic link.
-
-This option configures where this symbolic link is located. If the `file`
-location path points to a regular file, this setting has no effect
-(and ManageSieve cannot be used).
-
-##### Example
+#### Example
 
 ```
-plugin {
-   ...
-   sieve = file:~/sieve;active=~/.dovecot.sieve
-   sieve_default = file:/var/lib/dovecot/;name=default
+sieve_script personal {
+  driver = file
+  path = ~/sieve
+  active_path = ~/.dovecot.sieve
+}
+
+sieve_script default {
+  type = default
+  name = default
+  driver = file
+  path = /var/lib/dovecot/
```

Comment:
Wouldn't this usually be more like /etc/dovecot/sieve/default/

Reply:
changed